docs/content/tutorials/tutorial-streams.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: doc_page
+---
+
+## Load your own streaming data
+
+## Getting started
+
+This tutorial shows you how to load your own streams into Druid.
+
+For this tutorial, we'll assume you've already downloaded Druid as described in 
+the [single-machine quickstart](quickstart.html) and have it running on your local machine. You 
+don't need to have loaded any data yet.
+
+Once that's complete, you can load your own dataset by writing a custom ingestion spec.
+
+## Writing an ingestion spec
+
+When loading streams into Druid, we recommend using the [stream push](../ingestion/stream-push.html) 
+process. In this tutorial we'll be using [Tranquility Server](../ingestion/stream-ingestion.html#server) to push 
+data into Druid over HTTP.
+
+```note-info
+This tutorial will show you how to push streams to Druid using HTTP, but Druid additionally supports 
+a wide variety of batch and streaming loading methods. See the *[Loading files](batch-ingestion.html)* 
+and *[Loading streams](stream-ingestion.html)* pages for more information about other options, 
+including from Hadoop, Kafka, Storm, Samza, Spark Streaming, and your own JVM apps.
+```
+
+You can prepare for loading a new dataset over HTTP by writing a custom Tranquility Server 
+configuration. The bundled configuration is in `conf-quickstart/tranquility/server.json`, which 
+you can modify for your own needs.
+
+The most important questions are:
+
+  * What should the dataset be called? This is the "dataSource" field of the "dataSchema".
+  * Which field should be treated as a timestamp? This belongs in the "column" of the "timestampSpec".
+  * Which fields should be treated as dimensions? This belongs in the "dimensions" of the "dimensionsSpec".
+  * Which fields should be treated as measures? This belongs in the "metricsSpec".
```

Comment:
since the first bullet says "the "dataSource" field, I think the rest should not leave that out, so
'in the "column" field', 'in the "dimensions" field', 'in the "metricsSpec" section'.